openmc/geometry.py

```diff
@@ -257,7 +257,49 @@ def get_all_lattices(self):
                     lattices[cell.fill.id] = cell.fill
 
         return lattices
+    
+    def get_all_surfaces(self):
+        """
+        Return all surfaces used in the geometry
+
+        Returns
+        -------
+        collections.OrderedDict
+            Dictionary mapping surface IDs to :class:`openmc.Surface` instances
 
+        """
+        surfaces = OrderedDict()
+        
+        for cell in self.get_all_cells().values():
+            self._get_surfaces_from_region(surfaces, cell.region)
+        return surfaces
+    
+    def _get_surfaces_from_region(self, surfaces, region):
```

Comment:
Yeah, I agree most of this logic belongs in `region.py`. Are you suggesting we have a `Region.get_all_surfaces()` staticmethod? Or that each subclass of `Region` have its own `get_all_surfaces()` method?